src/internal/operations/backup_integration_test.go

```diff
@@ -1087,6 +1099,371 @@ func (suite *BackupOpIntegrationSuite) TestBackup_Run_oneDrive() {
 	runAndCheckBackup(t, ctx, &bo, mb)
 }
 
+func mustGetDefaultDriveID(
+	t *testing.T,
+	ctx context.Context, //revive:disable-line:context-as-argument
+	service graph.Servicer,
+	userID string,
+) string {
+	d, err := service.Client().UsersById(userID).Drive().Get(ctx, nil)
+	if err != nil {
+		err = graph.Wrap(
+			ctx,
+			err,
+			"failed to retrieve default user drive").
+			With("user", userID)
+	}
+
+	require.NoError(t, err)
+
+	id, ok := ptr.ValOK(d.GetId())
+	require.True(t, ok, "drive ID not set")
+	require.NotEmpty(t, id)
+
+	return id
+}
+
+// TestBackup_Run ensures that Integration Testing works for OneDrive
+func (suite *BackupOpIntegrationSuite) TestBackup_Run_oneDriveIncrementals() {
+	// TODO: Enable once we have https://github.com/alcionai/corso/pull/2642
+	suite.T().Skip("Enable once OneDrive incrementals is available")
+
+	ctx, flush := tester.NewContext()
+	defer flush()
+
+	var (
+		t    = suite.T()
+		acct = tester.NewM365Account(t)
+		ffs  = control.Toggles{}
+		mb   = evmock.NewBus()
+
+		// `now` has to be formatted with SimpleDateTimeOneDrive as
+		// some onedrive cannot have `:` in file/folder names
+		now = common.FormatNow(common.SimpleTimeTesting)
+
+		owners = []string{suite.user}
+
+		categories = map[path.CategoryType][]string{
+			path.FilesCategory: {graph.DeltaURLsFileName, graph.PreviousPathFileName},
+		}
+		container1 = fmt.Sprintf("%s%d_%s", incrementalsDestContainerPrefix, 1, now)
+		container2 = fmt.Sprintf("%s%d_%s", incrementalsDestContainerPrefix, 2, now)
+		container3 = fmt.Sprintf("%s%d_%s", incrementalsDestContainerPrefix, 3, now)
+
+		genDests = []string{container1, container2}
+	)
+
+	m365, err := acct.M365Config()
+	require.NoError(t, err)
+
+	gc, err := connector.NewGraphConnector(
+		ctx,
+		graph.HTTPClient(graph.NoTimeout()),
+		acct,
+		connector.Users,
+		fault.New(true))
+	require.NoError(t, err)
+
+	driveID := mustGetDefaultDriveID(t, ctx, gc.Service, suite.user)
+
+	fileDBF := func(id, timeStamp, subject, body string) []byte {
```

Comment:
> fileDBF

file Dirty Booger Flinger?

Reply:
I'm not actually sure on this one. The skeleton for this test is from the exchange incremental test and I'm pretty sure this is from there :P